src/script/cpp_api/s_player.h

```diff
@@ -38,6 +40,7 @@ class ScriptApiPlayer
 	void on_joinplayer(ServerActiveObject *player);
 	void on_leaveplayer(ServerActiveObject *player);
 	void on_cheat(ServerActiveObject *player, const std::string &cheat_type);
+	bool on_punchplayer(ServerActiveObject *player, ServerActiveObject *hitter, float time_from_last_punch, const ToolCapabilities *toolcap, v3f dir);
```

Comment:
That line is too long. Every line should be shorter than 70 chars, but its ok to have lines up to 89 chars.